pkgs/development/libraries/pipewire/default.nix

```diff
@@ -71,8 +71,6 @@
 }:
 
 let
-  mesonEnableFeature = b: if b then "enabled" else "disabled";
-
   self = stdenv.mkDerivation rec {
```

Comment:
you can drop the entire `let` scope: nothing here makes use of `self` being named.